### PR TITLE
not_a_token page 

### PR DESCRIPTION
**Pull request by octavioamu**
##### Description

Add front end page  /not_a_token  

I make a video because fullsize capture is without the elements (since the appear with the scroll of viewport) 
https://share.vidyard.com/watch/bUE87J9T4ZtC7MrgzcgAhL

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

UI
##### Refers/Fixes

Refs: #1443

### CODE REVIEW

File: app/app/urls.py [hunk 1]
```diff
@@ -189,6 +189,7 @@
     re_path(r'^about/?', retail.views.about, name='about'),
     re_path(r'^mission/?', retail.views.mission, name='mission'),
     re_path(r'^vision/?', retail.views.vision, name='vision'),
+    re_path(r'^not_a_token/?', retail.views.not_a_token, name='not_a_token'),
```
Comment (thelostone-mc) [inline app/app/urls.py]: lol

File: app/retail/templates/shared/nav.html [hunk 2]
```diff
@@ -26,7 +26,7 @@
       <img id="logo" src="{% static "v2/images/logo_large.png" %}" data-hover="{% static "v2/images/logo_med_hover.gif" %}" width="150" height="50" />
     {% endif %}
   </a>
-  {% if not request.path|is_in_list:"/,/contributor,/about,/mission,/vision,/results,/slack,/help,/how/funder,/how/contributor,/tools" %}
+  {% if not request.path|is_in_list:"/,/contributor,/about,/mission,/vision,/not_a_token,/results,/slack,/help,/how/funder,/how/contributor,/tools" %}
```
Comment (thelostone-mc) [inline app/retail/templates/shared/nav.html]: you would have to resolve the conflicts here 😅 
added a regex template tag to ensure this shows up only on relevant pages 
rebase and you'll see the changes :P

**Comment by thelostone-mc** [comment]
@octavioamu mind squashing commits + rebasing + resolving conflicts ?

File: app/app/urls.py (hunk 1 above)
Comment (octavioamu) [reply app/app/urls.py]: @thelostone-mc first one was "is_not_a_shitcoin" so "not_a_token" is great hahahaha

File: app/retail/templates/shared/nav.html (hunk 2 above)
Comment (octavioamu) [reply app/retail/templates/shared/nav.html]: hahaha I'm looking at it right now 2 new pages more and we will  going end with a 1mb page with that " if not request.path" hahah I will fix it

**Comment by octavioamu** [comment]
@thelostone-mc 
Great I will change it to path

About the sizes are the ones in the design file, I found it big too but that page also have only a few content. Tell me if we want to make it smaller. 

I can disabled that animations, Im not fan at all of scroll interactions but that lib was already used hahah Personally I prefer without animations 

I will fix the mobile view. 

Yep the ones I have added are compressed but I will check if the others are too

**Comment by thelostone-mc** [comment]
Let's do away with the animation in that case ! 
Hmm could we give it a shot with a smaller font size ? ( It's your call to make )

**Comment by octavioamu** [comment]
@thelostone-mc take a look, I think is better like this: 
localhost_8000_not_a_token

**Comment by octavioamu** [comment]
Also better with the title smaller too
localhost_8000_not_a_token 1

**Comment by octavioamu** [comment]
@thelostone-mc everything done! 

kapture 2018-08-02 at 13 50 56

**Pull request merged**